src/components/PageHeader/PageHeader.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import "./PageHeader.style.css";
+import add from "../../assets/add.svg";
+
+
+function PageHeader({setShowModal,headertext, btntext}) {
```

Comment:
we need to make the button optional if no text is passed don't render it.
also, variable names must be camelCase not just lowercase names.

Reply:
I will need to use Regex for camelCase?